### DISCUSSION TITLE
pandoc is becoming a hammer...

### DISCUSSION THREAD

**Discussion by avidseeker** [jgm/pandoc#10086] [Show and tell]
... and everything looks like a nail.

```sh
TITLE=$(pandoc -t rst f.md -o /dev/null \
	-L <(echo "function Meta(m) print(pandoc.utils.stringify(m.title)) end")
)
```

**Comment by jgm** [comment]
```
$ pandoc-lua <(echo "print(2 * math.pi^2)")
19.739208802179
```

**Comment by alerque** [comment]
If you're going to use a hammer, why beat around the bush?

* Do you need the shell `echo`? No.
* Do you need the Lua `print`? No.
* Do you need double quotes when singles will do? No.
* Do you need the parenthesis? No.
* Do you need so many spaces? No.

Just whack the nail.

```console
$ pandoc lua <<< '2*math.pi^2'
19.739208802179
```
